wgsl/index.bs

```diff
@@ -4033,82 +4033,158 @@ WGSL [=predeclared|predeclares=] an [=enumerant=] for each of the texel formats
 
 ### Sampled Texture Types ### {#sampled-texture-type}
 
-<pre class='def'>
-`texture_1d<type>`
-`texture_2d<type>`
-`texture_2d_array<type>`
-`texture_3d<type>`
-`texture_cube<type>`
-`texture_cube_array<type>`
-</pre>
-* `type` [=shader-creation error|must=] be `f32`, `i32`, or `u32`
+A <dfn noexport dfn-for='type'>sampled texture</dfn> is capable
+of being [=memory access|accessed=] in conjunction with a [=type/sampler=].
+It can also be accessed without the use of a sampler.
+Sampled textures only allow [=read accesses=].
+
+The [=texel format=] of the texture is defined in the {{GPUTextureBindingLayout}}.
```

Comment:
That's not right.

It's the given by the format attribute of the GPUTexture object.

Suggest something like this:

> The [=texel format=] is the {{GPUTexture/format}} attribute of the {{GPUTexture}} bound to the texture variable.
> WebGPU [=validating shader binding|validates=] compatibility between the texture, the {{GPUTextureBindingLayout/sampleType}} of the bind group layout, and the [=sampled type=] of the texture variable.

Where validating shader binding links to https://gpuweb.github.io/gpuweb/#abstract-opdef-validating-shader-binding